system_setup/ui/views/wslconfadvanced.py

```diff
@@ -107,12 +105,15 @@ def __init__(self, controller, configuration_data):
                 configuration_data.interop_appendwindowspath,
             'gui_theme': configuration_data.gui_theme,
             'gui_followwintheme': configuration_data.gui_followwintheme,
-            'legacy_gui': configuration_data.legacy_gui,
-            'legacy_audio': configuration_data.legacy_audio,
-            'adv_ip_detect': configuration_data.adv_ip_detect,
-            'wsl_motd_news': configuration_data.wsl_motd_news,
-            'automount': configuration_data.automount,
-            'mountfstab': configuration_data.mountfstab,
+            'interop_guiintegration': configuration_data
+            .interop_guiintegration,
```

Comment:
Is this syntax excepted? This is not common python, could we write it with an escape line rather? (same for all others)

Reply:
I have now updated it to a better and more unified style. Also this is one of the three acceptable syntax by flake8

Reply:
Can’t we have something like:
```
'interop_guiintegration': \
     configuration_data.interop_guiintegration,
```
Which would be more aligned with the style used in subiquity?

Reply:
Unfortunately, that one is the only one I am not able to change since all of the changes violate rules in `flake8`:
![image](https://user-images.githubusercontent.com/15316889/135979895-7a2ad986-60cb-46b6-b613-9eec21449ab3.png)
![image](https://user-images.githubusercontent.com/15316889/135979663-6a940990-a395-4854-9f5f-0271203c5436.png)
![image](https://user-images.githubusercontent.com/15316889/135979785-ce655abe-60ca-44d7-8437-923649b09795.png)
![image](https://user-images.githubusercontent.com/15316889/135980102-a9668335-8ed6-4a8a-bdcd-729e695912be.png)

Reply:
Indeed, we can use the noqa pragma, but anyway, all those use case will be moot if you can remove the stuttering with the dict

Reply:
Gotcha